images/linux/scripts/installers/firefox.sh

```diff
@@ -6,6 +6,13 @@
 
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/install.sh
+source $HELPER_SCRIPTS/os.sh
+
+# Update source (PPA) for Firefox for Ubuntu22 
+if isUbuntu22; then
+    add-apt-repository ppa:mozillateam/ppa
```

Comment:
Should we remove this ppa like we do for e.g. git.sh?

Reply:
Controversial moment. On the one hand, third-party ppas, even if they belong to the developers of the tool, can cause dependency problems. On the other hand, if Canonical's official policy is currently moving towards no support for Firefox products, then we will be forced to do so.

Reply:
The "Get-FirefoxVersion" function was updated to mention the source repository in the readme file.